client-app/src/Components/Login.js

```diff
@@ -54,67 +38,71 @@ const Login = props => {
     };
 
     return (
-        <div>
-            <form onSubmit={handleSubmit} className="my-5 container">
-                <div className="mb-3">
+        <div className="login-container">
+            <form onSubmit={handleSubmit} className="login-form">
+                <div className="form-group">
                     <label htmlFor="email" className="form-label">
                         Email address
                     </label>
                     <input
                         type="email"
                         className="form-control"
+                        name="email"
+                        id="email"
                         value={credentials.email}
                         onChange={onChange}
-                        id="email"
-                        name="email"
-                        aria-describedby="emailHelp"
                     />
-                    <div id="emailHelp" className="form-text">
+                    <small className="form-text">
                         We'll never share your email with anyone else.
-                    </div>
+                    </small>
                 </div>
-                <div className="mb-3">
+
+                <div className="form-group">
                     <label htmlFor="password" className="form-label">
                         Password
                     </label>
                     <input
                         type="password"
                         className="form-control"
-                        value={credentials.password}
-                        onChange={onChange}
                         name="password"
                         id="password"
+                        value={credentials.password}
+                        onChange={onChange}
                     />
                 </div>
-                <div className="mb-3">
+
+                <div className="form-group">
                     <label htmlFor="captcha" className="form-label">
                         CAPTCHA: {captcha}
                     </label>
                     <input
                         type="text"
                         className="form-control"
+                        name="captcha"
+                        id="captcha"
                         value={captchaValue}
                         onChange={handleCaptchaChange}
-                        id="captcha"
-                        name="captcha"
                     />
                 </div>
+
                 {errorMessage && (
                     <div className="alert alert-danger" role="alert">
                         {errorMessage}
                     </div>
                 )}
+
                 <button
                     type="submit"
```

Comment:
Consider disabling the submit button based on form validity, not just captchaValue.